modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/info_bar/end.jsp

```diff
@@ -28,10 +28,29 @@
 </div>
 
 <c:if test="<%= fixed %>">
-	<aui:script require="metal-affix/src/Affix">
-		new metalAffixSrcAffix.default({
-			element: '.info-bar-container',
-			offsetTop: 15,
-		});
+	<aui:script sandbox="<%= true %>">
+		const element = document.querySelector('.info-bar-container');
+
+		let lastPosition;
+		let topOffset = 15;
+
+		function checkPosition() {
+			let newPosition = 'affix';
+
+			if (document.defaultView.pageYOffset <= topOffset) {
+				newPosition = 'affix-top';
+			}
+
+			if (element && newPosition !== lastPosition) {
+				element.classList.add(newPosition);
+				element.classList.remove(lastPosition);
+
+				lastPosition = newPosition;
+			}
+		}
+
+		checkPosition();
+
+		document.addEventListener('scroll', checkPosition);
```

Comment:
This is a global listener, that will stay after we navigate away from the page. We need to remove it, for example like [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/document-library/document-library-analytics/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp#L67-L69).

Reply:
👍  I wasn't quite sure if we used that in a taglib or not, but I guess it makes sense since the taglib is used in a portlet.